test/controllers/api/v1/locations_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require 'test_helper'
+
+class Api::V1::LocationsControllerTest < ActionDispatch::IntegrationTest
+  test "POST /api/v1/code_schools/:code_school_id/locations" do
```

Comment:
Can you please add in the name of the method under test in each of these "test" descriptions.  i.e. ":create POST ....."

Reply:
done